.github/workflows/pull_request.yaml

```diff
@@ -9,38 +9,28 @@ on:
 
 env:
   CHART_NAME: "application"
-  KUBERNETES_VERSION: "1.18.0"
-  KIND_VERSION: "0.7.0"
+  OPENSHIFT_TEST_CLUSTER: "https://api.binero-test.8sdzwd1l.kubeapp.cloud:6443"
```

Comment:
@Nebojsa92 this should be read from organization secret

Reply:
fixed